models/public/yolo-v4-tf/yolo-v4-tf.md

```diff
@@ -0,0 +1,138 @@
+# yolo-v4-tf
+
+## Use Case and High-Level Description
+
+YOLO v4 is a real-time object detection model implemented with Keras\* from this [repository](https://github.com/david8862/keras-YOLOv3-model-set) and converted to TensorFlow\* framework. This model was pretrained on COCO\* dataset with 80 classes.
+
+## Conversion
+
+1. Download or clone the official [repository](https://github.com/david8862/keras-YOLOv3-model-set) (tested on `ffede5` commit).
```

Comment:
This commit seems suspect. The README file in this version doesn't even mention YOLOv4.